src/nyc_trees/apps/home/views.py

```diff
@@ -21,6 +24,28 @@ def retrieve_job_status(request):
     pass
 
 
+def redirect_to_flat_page(url):
```

Comment:
redirect is probably the wrong word. It does not perform an HttpResponseRedirect. It renders a flat page. Not a context, it returns rendered markup.